ts/src/bitmex.ts

```diff
@@ -1830,10 +1978,11 @@ export default class bitmex extends Exchange {
             }
         }
         const brokerId = this.safeString (this.options, 'brokerId', 'CCXT');
+        const qty = parseFloat (this.amountToPrecision (symbol, amount));
         const request = {
             'symbol': market['id'],
             'side': this.capitalize (side),
-            'orderQty': parseFloat (this.amountToPrecision (symbol, amount)), // lot size multiplied by the number of contracts
+            'orderQty': qty, // lot size multiplied by the number of contracts
```

Comment:
```suggestion
            'orderQty': parseFloat (this.amountToPrecision (symbol, amount)), // lot size multiplied by the number of contracts
```

maybe revert as it was before commit , would be one less change in diff.

Reply:
also note, down below several lines, under `editOrder` for this request param, you might also need to add  `parseFloat`

Reply:
btw, just for info, for such thing, i didnt make an override of `amountToPrecision` in bitmex class itself, instead i've added a new method named `convertToRawMarketQuantity` and used :  
```
'orderQty': this.convertToRawMarketQuantity (symbol, amount),
```

Reply:
@ttodua actually I was considering removing the `parseFloat` call and replace it with `parseToInt` or something, because using bitmex's precision I'm not sure if will end up sending a number with decimal points

Reply:
yes, but what I mean here is that the same should be done for editorder as in createOrder

Reply:
@ttodua they should be the same now